### ISSUE TITLE
Signal when annyang starts/stops listening

### ISSUE THREAD

**Issue by 4tee** [TalAter/annyang#157]
Hi, Thanks for this beautiful library. I am using your library with angularjs. I have a question. I noticed that android devices actually get an audio cue when annyang is listening, but desktop chrome browser is not. To get more accurate result, I like to display some visual cue when annyang has started/stopped listening. Any advice, please? Thanks.

**Comment by TalAter** [comment]
Hi @4tee 
You could [add a callback](https://github.com/TalAter/annyang/blob/master/docs/README.md#addcallbacktype-callback-context) to run a function and display some kind of indication when the `start` event fires.

You could also use [SpeechKITT](https://github.com/TalAter/SpeechKITT) to get visual indications like this. SpeechKITT is still under development, so I would love your feedback on what kind of visual indications you are looking for.

**Comment by 4tee** [comment]
Hi @TalAter , thanks for the reply. Not sure why I didn't get notification when you reply. So sorry for late reply. I ended up using `callback` as you mentioned. The only challenge was `start` event fired on first time and no more thereafter. Not sure if it is the right approach; I set the `start` function with `autoRestart: false, continuous: false` and manually call `start` function again whenever it stopped so that I can show/hide visual cue.

I had explored on SpeechKITT too. It requires the user to click on the button as a cue to start listening. The only interaction for my app is voice command and hence it is not suitable in my case. Thanks a lot for the reply.
